angular-poller.js

```diff
@@ -308,6 +308,19 @@
                 reset: function () {
                     this.stopAll();
                     pollers = [];
+                },
+
+                /**
+                * Stop and removes a specific poller service
+                */
+                remove: function (poller) {
```

Comment:
Here we go: https://github.com/emmaguo/angular-poller/commit/3831aadd5b287d1ce7d520f3fd20cd38af78ccc3. :-)